cmd/xeniad/tests/setup_test.go

```diff
@@ -56,7 +56,7 @@ func runTest(m *testing.M) int {
 	}
 	defer db.CloseMGO(tests.Context)
 
-	store, err := cayleyshelf.New(mongoURI.String())
+	store, err := cayleyshelf.New(mongoURI.String(), nil)
```

Comment:
You'll need to use the same "session name" as the mongo session that was registered, in this case `tests.TestSession`.